views/dashboard-application.hbs

```diff
@@ -125,163 +123,135 @@
-                                <option {{#if user.pref3AVAddOn}} selected{{/if}}>Audio + Visual Add-on</option>
-                                <option {{#if user.pref3TIBox}} selected{{/if}}>TI Box</option>
-                            </select>
-                        </div>
-                        <span class='row'></span>
-                        <div class='form-group col-md-3'>
-                            <a href="/boxes" target="_blank"><button class='file-upload'>
-                                    CLICK HERE TO SEE BOXES </button></a>
-                        </div>
-                        <span class='row'></span>
-                        <div class='form-group col-sm-10 col-md-5'>
-                            <label>If you are in a team, will you or your partner be receiving/picking up your team's box?</label>
-                            <select class='form-control' name='getsTeamBox'>
-                                <option {{#unless user.getsTeamBox}} selected{{/unless}}>I'm not in a team</option>
-                                <option {{#if user.getsTeamBox}} selected{{/if}}>I will</option>
-                                <option>My partner will</option>
-                            </select>
-                        </div>
-                        <div class='form-group col-sm-10 col-md-5'>
-                            <label>Are you located near UCLA and willing to come pick up your box?*</label>
-                            <select class='form-control' name='canPickUpBox'>
-                                <option {{#if user.canPickUpBox}} selected{{/if}}>YES</option>
-                                <option {{#unless user.canPickUpBox}} selected{{/unless}}>NO</option>
-                            </select>
-                        </div>
-
-                        <div class='form-group col-md-6 col-sm-10 short-answer'>
-                            <label>If not, what address would you like your box shipped to? Please include your <b>FULL</b> address.</label>
-                            <textarea class='form-control textarea' name='address' rows='6'>{{user.address}}</textarea>
+                        <div class="miscellaneous_icon col-md-5 col-sm-10 col-xs-10">
+                            <img src="/img/application/applicationicon.png" alt="Ideahacks Icon" width="50%">
                         </div>
                     </fieldset>
                 </div>
-                <!-- BUTTONS -->
-                <div class='buttons row'>
-                    <button class='submit-application'>SUBMIT</button>
-                </div>
-                <p class="submit-message" style='color: transparent;'></p>
             </div>
-
+            <!-- BUTTONS -->
+            <div class='submit'>
+                <button>SUBMIT</button>
+            </div>
+            <p class="submit-message" style='color: transparent;'></p>
```

Comment:
Can you add some padding after the submit button/message? Right now it's uncomfortably close to the bottom of the page